apps/linear_algebra/tests/test_halide_blas.cpp

```diff
@@ -303,7 +303,7 @@ int main(int argc, char *argv[]) {
             d.run_tests(size);
         }
     } else {
-        int size = 64 * 7;
+        int size = 768;
```

Comment:
64*7 = 448... what happened here? This is now crashing sdot on Windows for me. I get:
```
Testing sdot: Error: Output buffer result is accessed at 0, which is beyond the max (-1) in dimension 0
```

Reply:
...sounds like we still have a bug. :-) Dunno why Windows would be special though. What is get_host_target*(?